Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -453,9 +453,9 @@ void GCMemcardManager::DrawIcons()
 
     auto viewport = m_slot_table[slot]->viewport();
     auto row = m_slot_table[slot]->indexAt(viewport->rect().topLeft()).row();
-    auto maxTableIndex = m_slot_table[slot]->indexAt(viewport->rect().bottomLeft());
+    auto max_table_index = m_slot_table[slot]->indexAt(viewport->rect().bottomLeft());
```

Comment:
Since this commit is a fixup for your previous commit, squashing those would be welcome.

Reply:
isn't it up to the merger to squash the commits? i for sure can't do it afaik :)

Reply:
It's up to you. You can do it with `git rebase` + `git push -f`.

Reply:
We merge history as-is, we don't use squashing at this point.

Reply:
well, that was a whole lot of pain to rebase ( it was more then just rebase and push. djesus).
anyway, should be squashed now